cli/README.md

```diff
@@ -74,6 +74,10 @@ We love our bleeding edge tech. The amount of speed and, honestly, _fun_ that co
 
 The stated goal of `create-t3-app` is to provide the quickest way to start a new full-stack, typesafe web application. We take typesafety seriously in these parts as it improves our productivity and helps us ship fewer bugs. Any decision that compromises the typesafe nature of `create-t3-app` is a decision that should be made in a different project.
 
+### 4. Environment Variables
+
+Some frameworks and build tools, like Next.js, suggest that you commit `.env` files to your project and only store secrets in `.env.local` files. This is a bad idea. Never commit `.env` files to your project. Never! In a T3 project, all `.env` files are ignored by git and you should use `.env` to store your environment variables, **not** `.env.local`. This means a T3 app will work seamsly with Pisma instead of having to write `npx dotenv -e .env.local -- ` in front of every Prisma command. 
```

Comment:
```suggestion
Some frameworks and build tools, like Next.js, suggest that you commit `.env` files to your project and only store secrets in `.env.local` files. This is a bad idea. Never commit `.env` files to your project. Never! In a T3 project, all `.env` files are ignored by git and you should use `.env` to store your environment variables, **not** `.env.local`. This means a T3 app will work seamlessly with Prisma instead of having to write `npx dotenv -e .env.local -- ` in front of every Prisma command. 
```

Reply:
Is this really necessary to mention? Is anyone running Prisma commands with `npx dotenv -e .env.local -- ` before them?

Reply:
> Is this really necessary to mention? Is anyone running Prisma commands with `npx dotenv -e .env.local -- ` before them?

Yeah this is required if you use .env.local

Reply:
Sounds like an argument not to use .local 😂

Reply:
@juliusmarminge this is like 70% of my argument. Y'all made the right call, happy to die on this hill for you 🙏

Reply:
@juliusmarminge That's why I put it in, but I agree with @c-ehrlich that it's not a necessary argument to add in this case. 

For reference, prisma suggests this directly in their docs https://www.prisma.io/docs/guides/development-environment/environment-variables/managing-env-files-and-setting-variables#using-dotenv-cli-via-command-line